pkg/cli/login/loginoptions.go

```diff
@@ -174,6 +175,13 @@ func (o *LoginOptions) gatherAuthInfo() error {
 	// make a copy and use it to avoid mutating the original
 	t := *directClientConfig
 	clientConfig := &t
+	// dialToServer was called above but in case of user choosing insecure,
+	// need to call again for invalidServerURL
+	if clientConfig.Insecure {
+		if err := dialToServer(*clientConfig); err != nil {
```

Comment:
Move this into `getClientConfig` where `clientConfig.Insecure = true` is set.

Reply:
Moved, still need the 2nd call to dialToServer in case of clientConfig.Insecure modified mid-function, unless this check for text/html content-type belongs in the tokencmd.RequestToken() fn?  If so, we don't need the 2nd call, but flow continues to gatherAuthInfo().